docs/DEVELOPING.md

```diff
@@ -28,7 +28,7 @@ and another one to verify how it works in a more real environment as a parachain
 The following command will run the unit and integration tests:
 
 ```bash
-cargo +nightly test --workspace --release --features runtime-benchmarks,try-runtime
```

Comment:
Why is this removed? Is it only required for the node build?

Reply:
`--release` only for production nodes or benchmarks.
`+nightly` is set automatically by `rust-toolchain.toml`